src/notebook/example.md

```diff
@@ -143,3 +144,37 @@ query = 'SELECT * FROM "IDepartmentDynamic" WHERE department = 75'
 paris = pd.read_sql_query(query, engine)
 paris
 ```
+
+## Metabase API
+
+
+A large list of API endpoints are available ([doc](https://www.metabase.com/learn/administration/metabase-api)).
+See examples below.
+
+```python
+import requests
+import pandas as pd
+
+# admin metabase
+API_KEY = 'mb_dQBXZ4oMgf0bRsHY0zqlg6SSZuzoCqhitxX35KQI3hc='
+API_ADMIN_KEY = 'mb_eC8U3KEYG2SOD37Mw69NT9vbwnh78b5G/IcePXqKvoU=' 
```

Comment:
Those should be environment variables defined in the `env.d/notebook` file (see the `DATABASE_URL` example above).

Reply:
I add the variables in `env.d/notebook` but `os.getenv("API_ADMIN_KEY")` return None (i read that : ".env files are used to set the environment variable that are inserted in the generated docker images").

Reply:
Can you add your changes to this PR?

Maybe you forgot to restart the notebook container so that it refreshes environment variables as they are injected at startup.

Reply:
You are right, after `make run-notebook`, ADMIN_KEY is available.

-> I will add changes to the PR

Reply:
Changes are pushed (but there are five intermediate commits without inappropriate comment). 
Do you think i have to change the commit message (or squash) with `git rebase -i HEAD~6` ?

Reply:
Or we make a 'squash and merge' at the end of the PR ?